hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/di/LoggerConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package pl.allegro.tech.hermes.frontend.di;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggerConfiguration {
+    public static final String UNDERTOW_ERROR_RESPONSE_LOGGER = "io.undertow.request.error-response";
+
+    public static void disableResponseDebugLogger() {
```

Comment:
Method name indicates that it disables debug logger but it disables errors reponses. Maybe disableResponseErrorLogger would be better ?